test/site/_gods/bethany.md

```diff
@@ -7,3 +7,9 @@ name: Bethany
 And lo, so it came to pass, that Bethany ruined my birthday party.
```

Comment:
Creating a whole test site like this feels kinda heavy. Ideally you'd be testing components in isolation a little more - this feels like a very high-level integration test, and it may not be super helpful in pinpointing where something breaks.

Reply:
Agreed! My initial feeling was it'd be better to insert content dynamically into the jekyll process as individual markdown documents, but the `Jekyll::Site.read` step - which is the step before this plugin runs - seems to rely on reading files from a disk location.  I've reached out on the Jekyll community chat to see if there is a better way, and have read their repo's own tests but if there is a way, idgi.